x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/preview_histogram.tsx

```diff
@@ -206,12 +206,8 @@ export const PreviewHistogram = ({
             dataProviders,
             deletedEventIds,
             disabledCellActions: FIELDS_WITHOUT_CELL_ACTIONS,
-            // Fix for https://github.com/elastic/kibana/issues/135511, until we start writing proper
-            // simulated @timestamp values to the preview alerts. The preview alerts will have @timestamp values
-            // close to the server's `now` time, but the client clock could be out of sync with the server. So we
-            // avoid computing static dates for this time range filter and instead pass in a small relative time window.
             end: 'now+5m',
```

Comment:
This was `endDate` in an earlier commit, but now is reverted - is there a bug/reason we can't use `endDate` here instead of `now+5m`?

Reply:
Yes, it seems like there is a bug in the table/grid component. The alerts with the same timestamp as the `end` are not being included. Even `end: 'now+1s'` solves the issue. Could it be possible that the `end` value of the time range in table/grid is not inclusive?

Reply:
Done